# -*- coding: utf-8 -*-

from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_gider(object):
    def setupUi(self, gider):
        gider.setObjectName("gider")
        gider.resize(400, 275)
        self.label_6 = QtWidgets.QLabel(gider)
        self.label_6.setGeometry(QtCore.QRect(30, 40, 101, 41))
        font = QtGui.QFont()
        font.setFamily("Calibri")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label_6.setFont(font)
        self.label_6.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.label_6.setObjectName("label_6")
        self.label_7 = QtWidgets.QLabel(gider)
        self.label_7.setGeometry(QtCore.QRect(30, 90, 101, 41))
        font = QtGui.QFont()
        font.setFamily("Calibri")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label_7.setFont(font)
        self.label_7.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.label_7.setObjectName("label_7")
        self.label_8 = QtWidgets.QLabel(gider)
        self.label_8.setGeometry(QtCore.QRect(10, 140, 121, 41))
        font = QtGui.QFont()
        font.setFamily("Calibri")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.label_8.setFont(font)
        self.label_8.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.label_8.setObjectName("label_8")
        self.txt_tarih = QtWidgets.QDateEdit(gider)
        self.txt_tarih.setGeometry(QtCore.QRect(140, 50, 231, 31))
        self.txt_tarih.setObjectName("txt_tarih")
        self.txt_ad = QtWidgets.QLineEdit(gider)
        self.txt_ad.setGeometry(QtCore.QRect(140, 100, 231, 31))
        self.txt_ad.setObjectName("txt_ad")
        self.txt_miktar = QtWidgets.QLineEdit(gider)
        self.txt_miktar.setGeometry(QtCore.QRect(140, 150, 231, 31))
        self.txt_miktar.setObjectName("txt_miktar")
        self.btn_kaydet = QtWidgets.QPushButton(gider)
        self.btn_kaydet.setGeometry(QtCore.QRect(110, 220, 75, 23))
        self.btn_kaydet.setObjectName("btn_kaydet")
        self.btn_anamenu = QtWidgets.QPushButton(gider)
        self.btn_anamenu.setGeometry(QtCore.QRect(230, 220, 75, 23))
        self.btn_anamenu.setObjectName("btn_anamenu")

        self.retranslateUi(gider)
        QtCore.QMetaObject.connectSlotsByName(gider)

    def retranslateUi(self, gider):
        _translate = QtCore.QCoreApplication.translate
        gider.setWindowTitle(_translate("gider", "Gider Formu"))
        self.label_6.setText(_translate("gider", "Gider Tarihi:"))
        self.label_7.setText(_translate("gider", "Gider Adı:"))
        self.label_8.setText(_translate("gider", "Gider Miktarı:"))
        self.btn_kaydet.setText(_translate("gider", "Kaydet"))
        self.btn_anamenu.setText(_translate("gider", "AnaMenu"))

